PluginAppLoader/Apps/SettingsImportExport/manifest.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<packageManifest xmlns:cp="http://schemas.openxmlformats.org/package/2006/metadata/core-properties" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://schemas.robodk.com/package/2022/manifest" version="1.0">
+  <cp:coreProperties>
+    <cp:category>Application</cp:category>
+    <dcterms:created xsi:type="dcterms:W3CDTF">2024-04-18T11:58:15.425Z</dcterms:created>
+    <dc:identifier>app.settingsimport-export</dc:identifier>
+    <dc:language>en</dc:language>
+    <dcterms:modified xsi:type="dcterms:W3CDTF">2024-04-18T11:58:15.425Z</dcterms:modified>
+    <cp:revision>1</cp:revision>
+    <dc:title>Settings Import-Export</dc:title>
```

Comment:
Add a description like other addins: https://robodk.com/addins [The ... App for RoboDK..]